pkgs/by-name/fn/fna3d/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, SDL2
+}:
+stdenv.mkDerivation rec {
+  pname = "fna3d";
+  version = "23.11";
+
+  src = fetchFromGitHub {
+    owner = "FNA-XNA";
+    repo = "FNA3D";
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-UXphZQrk+MUOcUIqZ0ho3HTP5vNZjN9ifsANOJbTfVE=";
+  };
+
+  buildInputs = [ cmake SDL2 ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 libFNA3D.so $out/lib/libFNA3D.so
+    ln -s $out/lib/libFNA3D.so $out/lib/libFNA3D.so.0
+    ln -s $out/lib/libFNA3D.so $out/lib/libFNA3D.so.0.${version}
```

I think relative symlinks are more common here:

```suggestion
    ln -s libFNA3D.so $out/lib/libFNA3D.so.0
    ln -s libFNA3D.so $out/lib/libFNA3D.so.0.${version}
```